mcp3564.c

```diff
@@ -0,0 +1,286 @@
+/***********************************************************************************

```

Comment:
you changed only one line, but the PR contains two full extra files. If you remove them, I can merge the PR :)

Reply:
Whoops. Sorry. It's the first time I'm making a PR and I didn't expect the newly uploaded files to get into older PR.
All should be allright now.

Reply:
sweet. thanks for fixing it and submiting a PR